### PR TITLE
Potato battery nerf

### PR DESCRIPTION
**Pull request by Jalleo**
Potato batteries can no longer have a higher charge than bluespace batteries. They can be on par if you work hard enough.
:cl:
balance: nerfs power cells from a insane max possibility
/:cl:

### CODE REVIEW

**Comment by Jalleo** [comment]
dammit didnt remove the comments due to a mistake hold on

**Comment by arsserpentarium** [comment]
They should be higher, since 100 potency batteries is hard to obtain.

File: code/modules/hydroponics/plant_genes.dm [hunk 1]
```diff
@@ -212,7 +212,6 @@
 	// Cell recharging trait. Charges all mob's power cells to (potency*rate)% mark when eaten.
 	// Generates sparks on squash.
 	// Small (potency*rate*5) chance to shock squish or slip target for (potency*rate*5) damage.
-	// Multiplies max charge by (rate*1000) when used in potato power cells.
```
Comment (ChangelingRain) [inline code/modules/hydroponics/plant_genes.dm]: correct, don't remove

File: code/modules/hydroponics/plant_genes.dm [hunk 2]
```diff
@@ -355,8 +354,8 @@
 
 			// The secret of potato supercells!
 			var/datum/plant_gene/trait/cell_charge/CG = G.seed.get_gene(/datum/plant_gene/trait/cell_charge)
-			if(CG) // 10x charge for deafult cell charge gene - 20 000 with 100 potency.
-				pocell.maxcharge *= CG.rate*1000
+			if(CG) // Cell charge increases amount of power but not higher than bluespace
```
Comment (ChangelingRain) [inline code/modules/hydroponics/plant_genes.dm]: give actual numbers for the resulting maxcharge

File: code/modules/hydroponics/plant_genes.dm (hunk 1 above)
Comment (Jalleo) [reply code/modules/hydroponics/plant_genes.dm]: Fine I will tomorrow.

File: code/modules/hydroponics/plant_genes.dm (hunk 2 above)
Comment (Jalleo) [reply code/modules/hydroponics/plant_genes.dm]: See above.

**Comment by improvedname** [comment]
Put a I DED tag because he clearly died to 400K potato bombs

**Comment by Jalleo** [comment]
Actually no. I heard how much power it stores and went no that breaks all the balance by stupid o clock. I only found out by others salting over potato bombing.

**Comment by Dorsisdwarf** [comment]
Potato batteries being the best batteries is the best thing don't you dare make them suck. If you think 400k is too much half it or something don't take it all the way to where a completely perfected one is only as good as the ones R&D press 5 buttons for...

**Comment by Bombpants** [comment]
Reduce it like @Dorsisdwarf said, don't just nerf it back down to bluespace levels.

**Comment by PKPenguin321** [comment]
what exactly is so bad about them being exactly on par with bluespace batteries
you realize that's still really strong right

**Comment by arsserpentarium** [comment]
Dorisdwarf is right.

**Comment by 81Denton** [comment]
Potato batteries should clearly be better if you consider the amount of work involved. 
Not to mention that 9 out of 10 spessmen will still prefer the self-charging xenobio cells.

**Comment by Jalleo** [comment]
You all forget that by the point you are getting these you are likely manipulating the yield to get as many as possible. This still means you can easily fill a fuckton of SMES with these and have power for days.

**Comment by arsserpentarium** [comment]
Yes,mass production is strong side of botany. But why even bother if you can't get something more powerful?

**Comment by Tlaltecuhtli** [comment]
gotta nerf them unused features

**Comment by 81Denton** [comment]
>This still means you can easily fill a fuckton of SMES with these and have power for days.
You can do the same by hitting the 10x print button on the RnD protolathe - the cells are weaker in comparison, but there is practically no effort required.
Besides, station power is an absolute non-issue in its current state.

**Comment by Buggy123** [comment]
Let me repeat an argument I've made many, many times:
Everything is overpowered.

Every job and just about every mechanic is overpowered in its own way, and has the potential for abuse. If you want to nerf something because it's overpowered, then unless you are a myopic retard who can't see this fact, you should also want to nerf about _half of the contents of the entire game_. No, you don't nerf something because its overpowered, you nerf it because it gets abused and it needs to be stopped.

So, has anyone else noticed a rampant abuse of ultra-high capacity potato batteries lately? Because I sure haven't. I mean, with Goof's PR it could have become an issue, but that got reverted _quite_ quickly and probably isn't going to come up again. So, I think we're fine here, and this PR is unnecessary and uncalled for.

**Comment by Iamgoofball** [comment]
see I made the batteries use logarithmic explosions and scaled it so that the maxcap was 20k because I got told that 20k was as high as batteries went from potency

**Comment by Buggy123** [comment]
In fairness Goofball I actually like your PR and the idea of botany growing infinite maxcaps sounds really funny, but I can see that it might get abused pretty quickly, and the explosions of normal batteries would be too wimpy if the maxcap was scaled to 400k.

**Comment by Iamgoofball** [comment]
you misunderstand

I made it the OLD maxcap, 3/7/14, not the 5/10/20 one we had now, the point was to nerf it

**Comment by Buggy123** [comment]
Ah, okay thank you for clearing that up. Was a little confused about that. Still though, I find the idea of botany growing 5/10/20s en-mass kinda funny but prone to abuse, and I don't think it'd be very easy to balance such that 400ks produced a more reasonable explosion, without making normal sized batteries useless under the same scaling.

Really I need to look into the whole battery explosion thing more, barely know anything about it since its so astounding rare.

File: code/modules/hydroponics/plant_genes.dm [hunk 3]
```diff
@@ -212,7 +212,7 @@
 	// Cell recharging trait. Charges all mob's power cells to (potency*rate)% mark when eaten.
 	// Generates sparks on squash.
 	// Small (potency*rate*5) chance to shock squish or slip target for (potency*rate*5) damage.
-	// Multiplies max charge by (rate*1000) when used in potato power cells.
+	// Also affects plant batteries see capatative cell production datum
```
Comment (Jalleo) [inline code/modules/hydroponics/plant_genes.dm]: @ChangelingRain placed this comment here instead of a copypaste of the one that is at line 358

**Comment by arsserpentarium** [comment]
And note to you:
These comments wasn't bikeshredding. Why?
1. Bikeshredding is about of discussion of not very important details,instead of core needs.
It isn't about "don't fucking do it!or if you do,try to do it less".
2.because there aren't much important or not very important details in 3 lines of code.
3.don't fucking do it!or if you do,try to do it less.

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
Nothing should ever have a higher power than bluespace batteries. If someone wants to rescale them they need to be based on that principle.

**Comment by LetterN** [comment]
oh fuck not the kartoffer bombs

**Pull request merged**